src/ddmd/traits.d

```diff
@@ -370,6 +373,73 @@ extern (C++) Expression pointerBitmap(TraitsExp e)
     return ale;
 }
 
+extern (C++) Expression externDmangle(TraitsExp e, Scope* sc)
```

Comment:
In fact we're just prepending a qualified identifer (`_D3std3xml3bla...`), the type mangling already comes from the compiler. So it might be quite easy to do the replacements in druntime.